sdk/identity/identity/src/msal/utils.ts

```diff
@@ -74,11 +74,19 @@ export function getAuthority(tenantId: string, host?: string): string {
  * by sending it within the known authorities in the MSAL configuration.
  * @internal
  */
-export function getKnownAuthorities(tenantId: string, authorityHost: string): string[] {
+export function getKnownAuthorities(
+  tenantId: string,
+  authorityHost: string,
+  knownAuthorities?: string[]
+): string[] {
+  let allKnownAuthorities: string[] = [];
   if (tenantId === "adfs" && authorityHost) {
-    return [authorityHost];
+    allKnownAuthorities = [authorityHost];
```

Comment:
I think this is the only option we're going to have across all MSALs, i.e. setting the knownAuthories to [authorityHost]